blueprints/app/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - [main, master]
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 14.x
+
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.cache.outputs.dir }}
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
+    - name: Install Dependencies
+      run: <%= yarn ? 'yarn' : 'npm' %> install
+    - name: Lint
+      run: <%= yarn ? 'yarn' : 'npm' %> lint
```

Comment:
The `app` travis file doesn't have a listing step.  Do we want to keep this here?